libc/nt/master.sh

```diff
@@ -4828,7 +4828,7 @@ imp	'NtInitializeRegistry'					NtInitializeRegistry					ntdll		380
 imp	'NtInitiatePowerAction'					NtInitiatePowerAction					ntdll		381
 imp	'NtIsProcessInJob'					NtIsProcessInJob					ntdll		382
 imp	'NtIsSystemResumeAutomatic'				NtIsSystemResumeAutomatic				ntdll		383
-imp	'NtIsUILanguageComitted'				NtIsUILanguageComitted					ntdll		384
```

Comment:
Please revert changes to this file. This is the Windows API. We can't correct typos here.